src/main/java/com/example/clinic_management/mapper/AutoSymptomMapper.java

```diff
@@ -5,8 +5,6 @@
 import com.example.clinic_management.entities.Symptom;
 import org.mapstruct.Mapper;
 
-@Mapper(componentModel = "spring",
```

Comment:
no just delete "uses" as it not necessary, keep the componentModel to make it control by Spring framework

Reply:
oke